spec/features/12_user_uses_nav_bar_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature 'navbar has links to site features' do
+  let!(:user) { FactoryGirl.create(:user) }
+
+  scenario 'authenticated user clicks on nav bar links from any page' do
+    sign_in user
+    visit root_path
+    expect(page).to have_link 'Sign Out'
+    expect(page).to have_link 'Add a Figure'
+    expect(page).to have_link 'Figures'
+    expect(page).to have_link 'Profile'
+
+    click_link 'Profile'
+    visit root_path
+    expect(page).to have_link 'Sign Out'
+    expect(page).to have_link 'Add a Figure'
+    expect(page).to have_link 'Figures'
+    expect(page).to have_link 'Profile'
+
+    click_link 'Add a Figure'
+    visit root_path
+    expect(page).to have_link 'Sign Out'
+    expect(page).to have_link 'Add a Figure'
+    expect(page).to have_link 'Figures'
+    expect(page).to have_link 'Profile'
+  end
```

Comment:
you are really testing 3 different things! They should be 3 separate tests. Each time you go to a new page should be a new test in this case.